common/vpcclient/vpcfilevolume/expand_volume.go

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2021 IBM Corp.
```

Comment:
please update the copyright year to 2022, this is applicable to all new files

Reply:
done